packages/drivers/odsp-driver/src/snapshotRepresentation.ts

```diff
@@ -0,0 +1,486 @@
+/*!
```

Comment:
Please add reference to a ZipIt (I think that's the name) spec and any other interesting resources that might be useful.
We should see if it can be made public, as otherwise it's not very clear what all of this is about. Even if it's not, we need to make it clear here that there is some standard that this code needs to follow and thus structure can't be easily changed.

Also I feel like file name does not reflect what this is about. It has less to do with snapshot and more with specific serialization format and programming model exposing data in such format. So file should be named appropriately

Reply:
Yes, I will check it out whether we can make it public or not. It is a sharepoint doc.